packages/snackbar/README.md

```diff
@@ -0,0 +1,65 @@
+# React Snackbar
+
+A React version of an [MDC Snackbar](https://github.com/material-components/material-components-web/tree/master/packages/mdc-snackbar).
+
+## Installation
+
+```
+npm install @material/react-snackbar
+```
+
+## Usage
+
+### Styles
+
+with Sass:
+```js
+import '@material/react-snackbar/index.scss';
+```
+
+with CSS:
+```js
+import '@material/react-snackbar/dist/snackbar.css';
+```
+
+### Javascript Instantiation
+```js
+import React from 'react';
+import Snackbar from '@material/react-snackbar';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <Snackbar message="Click Me!" actionText="dismiss" />
+    );
+  }
+}
+```
+
```

Comment:
Can you also add documentation on how to use `getTimeoutMs`, `getCloseOnEscape`, and `isOpen`? Something like:

```
If you need to get the `timeoutMs`, `closeOnEscape`, or `open` value, then you can use a ref like so:

import React from 'react';
import Snackbar from '@material/react-snackbar';
 class MyApp extends React.Component {
  getSnackbarInfo = (snackbar) => {
    if (!snackbar) return;
    console.log(snackbar.getTimeoutMs());
    console.log(snackbar.isOpen());
    console.log(snackbar.getCloseOnEscape());
  }
  render() {
    return (
      <Snackbar
        message="Click Me!"
        actionText="dismiss"
        ref={this.getSnackbarInfo}
      />
    );
  }
}
```